### PR TITLE
Fix shell errorr due to missing double quotes

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
Error looks like:
/home/runner/work/_temp/55608d39-c27e-4b05-bdb8-6c3a00820a6f.sh: line 43: [: !=: unary operator expected

See #22290

### CODE REVIEW

File: .github/workflows/ci_macos_mu4.yml [hunk 1]
```diff
@@ -56,7 +56,10 @@ jobs:
     - name: "Configure workflow"
       env:
         pull_request_title: ${{ github.event.pull_request.title }}
-      run: |
+        SENTRY_SERVER_MU4_KEY: ${{ secrets.SENTRY_SERVER_MU4_KEY }}
+        SENTRY_SERVER_SANDBOX_KEY: ${{ secrets.SENTRY_SERVER_SANDBOX_KEY }}
+        SENTRY_PROJECT: ${{ github.event.inputs.sentry_project }}
+       run: |
```
Comment (igorkorsukov) [inline .github/workflows/ci_macos_mu4.yml:62]: This change needs to be removed, it leads to an error (I fixed it for the master)

File: .github/workflows/ci_windows_mu4.yml [hunk 2]
```diff
@@ -53,7 +53,10 @@ jobs:
       shell: bash
       env:
         pull_request_title: ${{ github.event.pull_request.title }}
-      run: |
+        SENTRY_SERVER_MU4_KEY: ${{ secrets.SENTRY_SERVER_MU4_KEY }}
+        SENTRY_SERVER_SANDBOX_KEY: ${{ secrets.SENTRY_SERVER_SANDBOX_KEY }}
+        SENTRY_PROJECT: ${{ github.event.inputs.sentry_project }}
+       run: |
```
Comment (igorkorsukov) [inline .github/workflows/ci_windows_mu4.yml:59]:
```
run: |
```
This change needs to be removed, it leads to an error (I fixed it for the master)

File: .github/workflows/ci_macos_mu4.yml (hunk 1 above)
Comment (Jojo-Schmitz) [reply .github/workflows/ci_macos_mu4.yml]: Oops, sorry

File: .github/workflows/ci_windows_mu4.yml (hunk 2 above)
Comment (Jojo-Schmitz) [reply .github/workflows/ci_windows_mu4.yml]: Sorry, fixed too now

**Review by igorkorsukov — APPROVED** [review]

**Pull request merged**